docs/vulnerability.md

```diff
@@ -0,0 +1,46 @@
+## Vulnerability Summary
+
+### Basic Grype Vulnerability Data Shape
+````json
+ {
+   "vulnerability": {...},
+   "relatedVulnerabilities": [...],
+   "matchDetails": [...],
+   "artifact": {...},
+   "language": "",
+   "licenses": [...],
+   "cpes": [...],
+   "purl": "",
+   "metadata": {...}
+````
+
+#### Vulnerability
+All information on the specific vulnerability that was directly matched on (e.g. ID, severity, CVSS score, fix information, links for more information).
+
+#### RelatedVulnerabilities
+Information pertaining to vulnerabilities found to be related to the main reported vulnerability. Maybe the vulnerability we matched on was a GitHub Security Advisory, which has an upstream CVE (in the authoritative national vulnerability database). In these cases we list the upstream vulnerabilities here.
+
+#### MatchDetails
+This section tries to explain what we searched for while looking for a match and exactly what details on the package and vulnerability that lead to a match.
+
+#### Artifact 
+This is a subset of the information that we know about the package (when compared to the [Syft](https://github.com/anchore/syft****) json output, we summarize the metadata section). 
+This has information about where within the container image or directory we found the package, what kind of package it is, licensing info, pURLs, CPEs, etc.
+
+#### Language
+This section tries to explain the language ecosystem where the vulnerability was found ex: `CVE-2015-5237` is a vulnerability related to the Golang protobuf library so the language in this case would be "go".
+
+#### Licenses
+This section tries to enumerate all discovered licenses found associated with the package described under the `Artifact` section.
+
+#### CPES
+This section lists all CPE used to matched against vulnerability data found in the grype vulnerability database.
+
+#### PURL
+
+#### Metadata
+
+#### Interesting subsections to investigate
```

Comment:
does this section need to be in this doc? this seems more related to the issue that this was on than this doc